pkg/vpcmodel/abstractVPC.go

```diff
@@ -67,8 +59,12 @@ func (n *VPCResource) SynthesisKind() spec.ResourceType {
 	return ""
 }
 
-func (n *VPCResource) ExtendedName(c *VPCConfig) string {
-	return n.ExtendedPrefix(c) + n.Name()
+func (n *VPCResource) NameForAnalyzerOut(c *VPCConfig) string {
+	prefix := ""
+	if c != nil && c.IsMultipleVPCsConfig {
```

Comment:
maybe the config type should have a method that encapsulates this concept of required prefix?